quickstarts/deploy-vm/vm-quickstart-template.json

```diff
@@ -0,0 +1,352 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualNetworks_GoQSVM_vnet_name": {
+      "defaultValue": null,
```

Comment:
>"defaultValue": null [](start = 6, length = 20)

Ate this lines with null values really needed?

Reply:
As far as I know, yes. This was a direct export from the CLI, so I erred on the side of not touching it. The only issue I can see potentially being a problem is the `osDisk_name` value. I'll try and run a test to see if it collides before re-requesting review.

Reply:
No collisions on the OSDisk name, we're good. ✅

Reply:
Excellent